### PR TITLE
Add ability to use custom gestures

### LINKED ISSUE #412: Short press not working
press for a short tap does not work, only a longer press changes the state(chartPressState), verify the Gesture.Pan() method

### PR DESCRIPTION
**Pull request by keithluchtel**
### Description

Fixes #412 

#### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Checklist: (Feel free to delete this section upon completion)

- [x] I have included a [changeset](../CONTRIBUTING.md#changesets) if this change will require a version change to one of the packages.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have run `yarn run check:code` and all checks pass
- [x] I have created a changeset for new features, patches, or major changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by keithluchtel** [comment]
@zibs, @carbonrobot -- looking for some feedback on adding this functionality / API change to the proj. Sometimes we get requests about supporting new gesture types, and it's hard to account for everything internally.

File: lib/src/cartesian/CartesianChart.tsx [hunk 1]
```diff
@@ -478,12 +499,15 @@ export function CartesianChart<
     </Canvas>
   );
 
+  let composedGestures = customGestures ?? Gesture.Race();
+  if (chartPressState) {
+    composedGestures = Gesture.Race(composedGestures, panGesture);
+  }
```
[the remark this answers is not in the document]
Comment (keithluchtel) [reply lib/src/cartesian/CartesianChart.tsx:520]: This was due to removing the `chartPressState` prop in the getting started example. It should work now.

File: website/docs/cartesian/cartesian-chart.md [hunk 2]
```diff
@@ -192,6 +192,39 @@ The `onChartBoundsChange` prop is a function of the shape `onChartBoundsChange?:
 
 The `gestureLongPressDelay` prop allows you to set the delay in milliseconds before the pan gesture is activated. Defaults to `100`.
 
+### `customGestures`
+
+The `customGestures` prop allows you to provide custom gesture handlers that will work alongside (or instead of) the default chart press gestures. It accepts a `ComposedGesture` from react-native-gesture-handler.
+
+When both `customGestures` and `chartPressState` are provided, the gestures will be composed using `Gesture.Race()`, allowing either gesture to be active.
```
Comment (zibs) [inline website/docs/cartesian/cartesian-chart.md:199]: Let's add an inline example like from the `custom-gesture` file above:

```tsx
  const tapGesture = Gesture.Tap().onStart((e) => {
    state.isActive.value = true;
    ref.current?.handleTouch(state, e.x, e.y);
  });
  const composed = Gesture.Race(tapGesture);

```

**Review by zibs — APPROVED** [review]
Looks great!

**Pull request merged**